babel.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = (api) => {
 
   return {
     presets: [
-      ['@babel/preset-env', { targets: isTestEnv ? 'current node' : '> 1% or IE 11' }],
+      ['@babel/preset-env', { targets: isTestEnv ? 'current node' : undefined }],
```

Comment:
would it make sense to define a `targets` so we can declare this once?

```js
const targets = isTestEnv ? 'current node' : undefined;
...
      ['@babel/preset-env', { targets }],
```

Reply:
Yeah, I was wondering if there might be a way to simplify / share some of this configuration.

The challenge comes from the other `targets` configuration below, targeting our modern polyfills:

https://github.com/18F/identity-idp/blob/4eba3d16534553d2467933e89ff96d0619731056/babel.config.js#L20

So while `isTestEnv ? 'current node' : undefined` is sorta the "default" `targets` configuration, it's not the only one.

I also don't love how the "modern polyfills" configuration reimplements part of the `.browserslistrc` configuration. It would be nicer as something like...

```js
`(${config}) and supports es6-module`
```

Reply:
I think it's probably best to treat the "default" `targets` as the desirable (post-Internet Explorer) targets configuration, so consolidated in [86fe2bd](https://github.com/18F/identity-idp/pull/6154/commits/86fe2bd93a75c198881fb4cf01c90ed8a813f4d0).